### DISCUSSION TITLE
How do I make SE more precise in putting subtitles in proper place?

### DISCUSSION THREAD

**Discussion by cvrle77** [SubtitleEdit/subtitleedit#8051] [General]
![audio](https://github.com/SubtitleEdit/subtitleedit/assets/16167396/082cc9b5-0be3-46f7-a392-f28f43218166)
As can be seen from the image, SE doesn't place subtitle at the beginning of the speech sometimes, and sometimes it does. It is more or less the same story for ending of subtitle (it ends too soon sometimes)

Sometimes there's a longer speech that ends up as two subtitles both having two rows, but one is 5 seconds, and the other is 2, meaning, first timing is too long and second becomes too short.

Is it possible to improve precision? Can it be done through options, switches? Is this language model dependent, or SE does post-processing of timings?

**Comment by niksedk** [comment]
Are you using Whisper?

**Reply by cvrle77** [reply]
That is correct, I am using Purfview's Faster-Whisper

**Comment by cvrle77** [comment]
Redialing on this one, it seems that connection broke 😁

**Comment by niksedk** [comment]
It's a known issue that Whisper is not very precise, see e.g. https://github.com/openai/whisper/discussions/139

Note that Purfview's Faster Whisper has implemented some improvements regarding this!
Also, "Auto adjust timings" might improve timings a bit, but only with clear audio without background noise.

I think you waveform with text looks pretty okay. You could perhaps extend all cues duration by 50 ms.
